src/openapi3/handleJson.ts

```diff
@@ -74,6 +74,8 @@ const buildResourceFromSchema = (
               (property.items as OpenAPIV3.SchemaObject).format
             )
           : null,
+      enum: property.enum ? Object.values(property.enum) : null,
```

Comment:
Why `Object.values`?

Reply:
It was failsafe calling to ensure we always get array, I took it back and adjusted test case to work without it.  

I have notice that jsonref `parse` called here https://github.com/api-platform/api-doc-parser/blob/f63b172aed56935be1bdf6aeaa74e7e6cecda5d8/src/openapi3/handleJson.ts#L131 adds private `Symbol()` property (?), for example:
``` js
bookFormat: {
        type: 'string',
        description: 'The publication format of the book.',
        enum: [
          'AudiobookFormat',
          'EBook',
          'Paperback',
          'Hardcover',
          [Symbol()]: [Object]
        ],
        [Symbol()]: {
          registry: [Object],
          refs: [Set],
          parent: [Circular *1],
          root: [Object],
          scope: 'https://demo.api-platform.com/#/components/schemas/Book-book.read/properties/bookFormat',
          derefd: true
        }
      },
```
so tests failed before because `JSON.stringify` treats it as an object and gives:
`{"0": "AudiobookFormat", "1": (...) }`. I have just included it in a test case.